/**:
  ros__parameters:
    class_names: ["little", "mid", "big"]
    rename_car_to_truck_and_bus: true
    top2Ground: [0.998531, 0.000016, 0.054174, 0.000000,
                                  0.000016, 1.000000, -0.000583, 0.000000,
                                  -0.054174, 0.000583, 0.998531, 3.237808,
                                  0.000000, 0.000000, 0.000000, 1.000000]
